test/WebSites/ActionResultsWebSite/Controllers/ActionResultsVerificationController.cs

```diff
@@ -108,5 +119,20 @@ private DummyClass CreateDummy()
                 SampleString = "Foo"
             };
         }
+
+        private DisposableDummy CreateDisposableDummy()
+        {
+            return new DisposableDummy();
+        }
+
+        private class DisposableDummy : IDisposable
+        {
+            public static int disposeCallCount = 0;
```

Comment:
Not sure what other state to use. But this is a functional test, I guess its okay to use static? Or did you mean don't use static for any any kind of test?